src/foam/nanos/so/SystemOutageTask.js

```diff
@@ -31,6 +54,36 @@ foam.INTERFACE({
     {
       name: 'deactivate',
       args: 'X x'
+    },
+    {
+      name: 'cleanUp',
+      args: 'X x',
+      documentation: 'work to be done when task is removed',
+      javaCode: `
+        deactivate(x);
```

Comment:
Should we test SystemOutage.active first
so if active, call deactivate, else it should have already been called.